src/utils/contentstack.ts

```diff
@@ -85,3 +85,62 @@ const isEmptySEO = (seo: { [key: string]: any }) => {
 
     return true;
 };
+
+export const convertCSMarkdownToGeneralMarkdown = (cs_markdown: string) => {
```

Comment:
@jackyw2018 : Can I get a test case for this one. How are you identifying the citations (meaning numbered references)

Reply:
Do you mean to write a test or provide a test case?

I have manually tested by creating content on Contentstack UI and rendering it on DevCenter UI. 

Here is a [CS entry](https://app.contentstack.com/#!/stack/blt39790b633ee0d5a7/content-type/articles/en-us/entry/blte3314a3801e3f4a8/edit?branch=main) which I have:
1. CS-inserted Images in the middle and at the end
2. CS-inserted Link at the end

This appears to render correctly via the preview page.